yt_dlp/extractor/crunchyroll.py

```diff
@@ -739,7 +739,7 @@ def _get_params(self, lang):
                 headers={
                     'Authorization': auth_response['token_type'] + ' ' + auth_response['access_token']
                 })
-            cms = traverse_obj(policy_response, 'cms_beta', 'cms')
+            cms = policy_response.get('cms_web')
```

Comment:
Personally, I'd do this just in case

```suggestion
            cms = traverse_obj(policy_response, 'cms_web', 'cms_beta', 'cms')
```

Reply:
The implementing the suggested change works for me! I'm getting successful downloads now.

Can't wait for this PR to be officially done :grin: 

Thanks for all the hard work folks! :heart:

Reply:
That seems more likely to make bugs harder to understand than to actually help, to me. I only vaguely remember what the issue was that caused it to be switched to `traverse_obj` in the first place, though. Was it a regional difference in the api? Then *maybe* the fallbacks are a good idea, but I'd rather not implement them blindly. Instead, lets wait and see if anyone reports an issue with just using `cms_web`.

Reply:
The issues with #4898 also seem to disappear. Can @krandor confirm?

Reply:
> The implementing the suggested change works for me! I'm getting successful downloads now.

This seems quite confusing. Considering the output you pasted earlier, you're not even reaching the part of the code this change affects, so changing it shouldn't fix your problem...

Reply:
Based on what @winterbird-code wrote below, it sounds like @krandor's issue is actually a transient one, so it wasn't this change that fixed it, just time.

Reply:
> The issues with https://github.com/yt-dlp/yt-dlp/issues/4898 also seem to disappear. Can @krandor confirm?

Sorry for the late reply, but no, I'm still getting those errors here and there, but I think it might be a region or ISP issue. If I VPN to a VPS I have I'm able to download the episodes without issue.